lib/Echidna/Exec.hs

```diff
@@ -104,8 +103,11 @@ execTx = execTxWith vmExcept $ liftSH exec
 type PC = Int
 -- Index per operation in the source code, obtained from the source mapping 
 type OpIx = Int
+-- Stack size from the EVM
+type NumberFrames = Int 
```

Comment:
`FrameNumber` maybe?

Reply:
It's the length of the `evm . frames` list.

Reply:
`FrameCount` then?

Reply:
Done.